project.clj

```diff
@@ -1,15 +1,25 @@
 (defproject witan.workspace "0.1.0-SNAPSHOT"
   :description "FIXME: write description"
-  :url "http://example.com/FIXME"
-  :license {:name "Eclipse Public License"
-            :url "http://www.eclipse.org/legal/epl-v10.html"}
   :dependencies [[org.clojure/clojure "1.7.0"]
                  [org.clojure/core.async "0.2.371"]
                  [com.stuartsierra/component "0.3.1"]
+                 [com.taoensso/timbre "4.3.1"]
```

Comment:
would be good to look into how to get logstash format out of timbre if it is your logging framework of choice